gourmet/tests/test_exporters.py

```diff
@@ -29,7 +29,7 @@
 if not os.path.exists(OUTPUT_DIRECTORY): os.makedirs(OUTPUT_DIRECTORY)
 
 def confirm_strings_are_in_file (ss, fi):
-    whole_file = file(fi,'r').read()
+    whole_file = open(fi,'r').read()
```

Comment:
this file is never closed. I think it's fine to do something like

```python
with open(fi, 'r') as fp:
    whole_file = fp.read()
```

Reply:
You're right. Thanks!
Fixed